lib/App/Mi6.pm6

```diff
@@ -121,6 +121,7 @@ sub test(@file, Bool :$verbose, Int :$jobs) {
             @file = <t xt>.grep({.IO.d});
         }
         my @command = "prove", "-e", $*EXECUTABLE, |@option, |@file;
+        @command.unshift( "cmd.exe", "/c" ) if $*DISTRO.is-win;
         note "==> Set PERL6LIB=%*ENV<PERL6LIB>";
         note "==> @command[]";
         my $proc = mi6run |@command;
```

Comment:
I think mi6run takes care of that.

Reply:
Oh, yes. My bad. I was working on the version `zef` installed for me, it seems to be a bit behind.
Looking at the new code, yes that should do it, however the /x and /d flags are superfluous (and confusing)

/x is not a valid argument at all, and /d deactives autoruns.

All you need is /c

> ~>cmd.exe /?
Startet eine neue Instanz des Windows Befehlsinterpreters.

> 
> CMD [/A | /U] [/Q] [/D] [/E:ON | /E:OFF] [/F:ON | /F:OFF] [/V:ON | /V:OFF]
>     [[/S] [/C | /K] Zeichenfolge]
> 
> /C      Führt den Befehl in der Zeichenfolge aus und endet dann.  <-- That's what you want